### PR TITLE
Show exposed items after they receive an update

### PR DESCRIPTION
**Pull request by hakam80**
Signed-off-by: Hakam Saffour <hakam.saffour@gmail.com>

### CODE REVIEW

**Comment by hakam80** [comment]
This info would save a lot of time for debugging to find out why the item is not displayed in myopenhab!

**Comment by Confectrician** [comment]
Hey @hakam80, 

First:
Thanks for your contribution here. 👍 
That sounds really useful.

#### Sign-off

Due to the Contribution Guidelines that are valid here, you would have to `sign off` your work.
See https://github.com/openhab/openhab-docs/blob/gh-pages/CONTRIBUTING.md for further information.

As far as i know, it would be ok, to edit your Pull Request message, with your sign-off line in this case for such a small change, but maybe @ThomDietrich can say a bit more about this when he reads this.

Regards
Confectrician

**Comment by hakam80** [comment]
Hi @Confectrician 
I am not sure how to sign off

**Comment by Confectrician** [comment]
Hi,

### General way of signing off
The `sign-off` should be in the "commit message" that you can enter while submitting your changes.
(I think you are using the GitHub build in Editor currently?)

If yes, this would look a bit like this (sorry for the quick n' dirty screenshot):
image

1. The Editor with the changes you have made
2. The Commit Title (I have set it to yours from above to make the example clear)
3. Here's where the sign off should usually go

**But that's only fur the future, if you like to contribute another thing later and for correcting this pull request in the `2nd Step` i have written down below.**

## For this Pull Request:
**1st Step:**

Editing this part should be fine:
image

You should have a pencil or a similar symbol to edit this.
Put in your `sign off` there, to connect it to the whole pull request you made.

**2nd step:**
Please remove the `sign-off` you made in the `README.md`  with another commit and use the general way i showed you above.


## If something isn't clear yet:
Feel free to add a comment here and i will try to solve and answer your questions. 😄

**Comment by hakam80** [comment]
Hi @Confectrician
Thanks for the clarifications :)

**Comment by ThomDietrich** [comment]
Hey guys and also thanks from me for adding this detail @hakam80.

@Confectrician perfect explanation and that was definitely needed. Both of you however missed one tiny detail: https://github.com/openhab/openhab-docs/blame/gh-pages/_addons_ios/openhabcloud/readme.md#L11


Many parts of the documentation are imported from other openHAB repositories, e.g., the Binding- or add-on specific repository. This way we ensure that the articles are always in sync with the actual source code.
Long story short, you'll need to propose your change in the correct repository: 
https://github.com/openhab/openhab2-addons/blob/master/addons/io/org.openhab.io.openhabcloud/README.md

Thanks again for your contribution!

Please close this PR after you've created the new one.

**Comment by Confectrician** [comment]
> Both of you however missed one tiny detail

Yeah, reading html comments is one of my waterloo's, but i am working on it. 😄
